backend/services/api-test/src/main/java/io/metersphere/api/service/ApiExecuteResourceService.java

```diff
@@ -98,23 +94,4 @@ private void updateRunningReportStatus(GetRunScriptRequest request) {
             default -> throw new MSException("不支持的资源类型: " + request.getResourceType());
         }
     }
-
-    private void updateTaskRunningStatus(String taskId) {
-        ExecTask execTask = new ExecTask();
-        execTask.setId(taskId);
-        execTask.setStartTime(System.currentTimeMillis());
-        execTask.setStatus(ExecStatus.RUNNING.name());
-        execTaskMapper.updateByPrimaryKeySelective(execTask);
-    }
-
-    private void updateTaskItemRunningStatus(GetRunScriptRequest request) {
-        TaskItem taskItem = request.getTaskItem();
-        // 更新任务项状态
-        ExecTaskItem execTaskItem = new ExecTaskItem();
-        execTaskItem.setId(taskItem.getId());
-        execTaskItem.setStartTime(System.currentTimeMillis());
-        execTaskItem.setStatus(ExecStatus.RUNNING.name());
-        execTaskItem.setThreadId(request.getThreadId());
-        execTaskItemMapper.updateByPrimaryKeySelective(execTaskItem);
-    }
 }
\ No newline at end of file
```

Comment:
Code is mostly fine here except a couple things:

- Line `@Resource` for the mapper should be placed inside each resource annotation (execTaskItemMapper, execTaskMapper).

- Some imports and methods have been duplicated.

Optimization suggestions:

1. Remove any redundancies or unnecessary code.
Please correct these issues before running further processing. Please let me know what you'd like to do with this code next. It may need additional checks such as type validation on variables.